server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -886,6 +888,14 @@ protected Node(
 
             final RecoveryPlannerService recoveryPlannerService = getRecoveryPlannerService(threadPool, clusterService, repositoryService);
 
+            final HealthService healthService = new HealthService(
+                pluginsService.filterPlugins(HealthIndicatorPlugin.class)
+                    .stream()
+                    .map(HealthIndicatorPlugin::getHealthIndicatorServices)
+                    .flatMap(List::stream)
+                    .collect(toList())
```

Comment:
NIT: I would suggest to replace `.collect(toList())` with `.toList()`. It is shorter and also produces immutable list